tensorflow/lite/micro/kernels/softmax.cc

```diff
@@ -29,22 +29,39 @@ namespace micro {
 namespace activations {
 namespace {
 
+// Softmax parameter data that persists in user_data
+struct SoftmaxOpData {
+  struct SoftmaxParams params = {};
+  float table[256]{};
+  const int size_of_lut = 513;
```

Comment:
Can you make this constexpr instead of const?

Reply:
Done